woocommerce/woocommerce-admin
# Add padding on table header button

Fixes #7198 

Add padding on table header button

### Screenshots
Before
<img width="1120" alt="螢幕快照 2021-06-18 下午2 45 12" src="https://user-images.githubusercontent.com/56378160/122783948-06671d80-d280-11eb-9c31-6769d03a0a44.png">

After
<img width="1121" alt="螢幕快照 2021-06-18 下午2 46 11" src="https://user-images.githubusercontent.com/56378160/122783978-0d8e2b80-d280-11eb-8d0d-bb5f72e10f84.png">


### Detailed test instructions:

- Create tax rates
- Go to Analytics -> Taxes
- Check it the table style renders as expected
